src/app/design/styles/spacing/ux-styles-spacing.component.html

```diff
@@ -0,0 +1,519 @@
+		</div>
+	</sky-column>
+	<sky-column screenSmall="6">
+		<p class="sky-emphasized">Two column</p>
+		<div class="screenshot screenshot-info">
+			<img src="~/assets/img/guidelines/spacing/page-two-column-desktop.png" style="width: 100%;" />
+		</div>
+	</sky-column>
+</sky-row>
+	
+<h3>Containers</h3>
+<div>
+	<p>
+		Containers require padding to set their content away from the container edge. The amount of padding differs based on the size of the container. In general, SKY UX components handle padding for you.
+	</p>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="border: 15px solid #c1e8fb; box-sizing: border-box; width: 50px; height: 50px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Large containers often run near the edge of the viewport, so significant padding is necessary to prevent the content from feeling crunched. Examples include:</div>
+        <ul>
+          <li>Pages</li>
+          <li><a routerLink="/components/modal">Modals</a></li>
+					<li><a routerLink="/components/tile">Tiles</a></li>
+					<li><a routerLink="/components/sectioned-form">Sectioned forms</a></li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><sky-code>class="sky-padding-even-large"</sky-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>padding: 15px</div>
+      </div>
+    </div>
+	</div>
+	<div class="definition-table definition-table-wide">
+		<div class="definition-item">
+      <div class="sky-subsection-heading" style="border: 10px solid #c1e8fb; box-sizing: border-box; width: 50px; height: 50px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Small containers within larger containers require less padding to set the content apart from its surroundings because the larger containers already have padding. Examples include:</div>
+        <ul>
+          <li><a routerLink="/components/alert">Alerts</a></li>
+					<li><a routerLink="/components/card">Cards</a></li>
+					<li><a routerLink="/components/popover">Popovers</a></li>
+					<li><a routerLink="/components/repeater">Repeaters</a></li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><sky-code>class="sky-padding-even-default"</sky-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>padding: 10px</div>
+      </div>
+    </div>
+	</div>
+</div>
+	
+<h3>Content</h3>
+<div style="margin-bottom: 20px;">
+	<div class="sky-subsection-heading">Vertically arranged content</div>
```

Comment:
Not sure if it matters, but in the default theme we place Vertically arranged content before Horizontally arranged content, but it's in the opposite order in Modern theme.

Reply:
updated